.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: java
+
+jdk:
+  - oraclejdk8
+  - openjdk11
+
+cache:
+  directories:
+  - $HOME/.m2
+  - $HOME/.sonar/cache
+
+before_script:
+  - pip install --user codecov
+
+before_install:
+  - echo -n | openssl s_client -connect scan.coverity.com:443 | sed -ne '/-BEGIN CERTIFICATE-/,/-END CERTIFICATE-/p' | sudo tee -a /etc/ssl/certs/ca-
```

Comment:
Not necessary, we don't use coverity here.

Reply:
Removed